src/components/layout/dashboard/sideNav/ItemCategory.tsx

```diff
@@ -16,14 +16,21 @@ const ItemCategory = <T extends object>({ item, first }: Props<T>) => {
 
   // If all children in category are hidden, hide parent
   if (childItems.length === 0) return null;
+
   return (
     <ItemBase
       item={item}
       renderTitle={(title) => (
         <Typography
-          variant='body2'
+          variant='caption'
           color='text.secondary'
-          sx={{ pt: first ? 0 : 2, px: 2, textTransform: 'uppercase' }}
+          display='block'
+          sx={{
+            px: 1,
+            mt: first ? -1 : 1,
```

Comment:
For future clarity, I tend to prefer positive margins where they are needed, instead of negative margins to cancel them out only where they aren't. But, feel free to ignore this if it's a significant refactor. I gave it a quick shot and it didn't seem simple

Reply:
agreed, there was a fair amount of complexity there. this isn't ideal but at least this isn't code that is touched frequently